internal/client/v1/client/vald/vald.go

```diff
@@ -61,6 +65,10 @@ func New(opts ...Option) (Client, error) {
 	return c, nil
 }
 
+func NewValdClient(cc *grpc.ClientConn) Client {
```

Comment:
It is hard to differentiate the origin client as this function name is hard to understand...
```suggestion
func NewSingleConnectionClient(cc *grpc.ClientConn) Client {
```

Reply:
I also suggest to separate this single connection used client implementation to another file

Reply:
I guess I'll have to define Interface itself in a separate file to accept your opinion.

Would it be so hard to read just 4 methods?

Reply:
OK. I accept your opinion. 
but can you review the above code suggestion to change the func name?

Reply:
I also thought about the name of the construct method, but I don't think it's a problem since the name and I/F are aligned with this in terms of the grpc-go constructor.